### PR TITLE
Vcd 10.1 updates part4

### PR DESCRIPTION
**Pull request by suehighmoor**
Next batch of updates for VCD 10.1 (a lot of these are just replacing instances of vCloud Director with VMware Cloud Director)

- Bring Your Own Firewall - Known working and non-working firewalls
- Bring Your Own Firewall Service Scope
- How to access UKCloud services from Walled Garden and Secure Remote Access
- Janet connectivity FAQs
- Disaster Recovery as a Service Service Scope
- Getting Started Guide for UKCloud for Oracle Software
- Understanding your invoice evidence file
- How to configure a Distributed Firewall
- How to configure a Distributed Logical Router
- How to perform a failover
- How to change IPsec VPN settings via the vCloud API
- Fixing your IPsec VPN following upgrade to NSX 6.4
- Fixing your advanced gateway if you encounter NullPointerException error

### CODE REVIEW

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

File: articles/draas/draas-sco.md [hunk 1]
```diff
@@ -113,23 +113,23 @@ You can self-manage your protected environment via the Zerto Self-Service Portal
 
 - Perform live failovers
 
-We've disabled some areas of the Zerto Self-Service Portal as they're for services we don't presently support. These include:
+We've disabled some areas of the ZSSP as they're for services we don't presently support. These include:
```
Comment (gsmith101) [inline articles/draas/draas-sco.md:116]: The Zerto Self-Service Portal (ZSSP)

File: articles/draas/draas-sco.md [hunk 2]
```diff
@@ -113,23 +113,23 @@ You can self-manage your protected environment via the Zerto Self-Service Portal
...
 
 - VMs with virtual disks with individual disks greater than 3TB
 
-- VMs which are clustered using shared virtual disks
+- VMs that are clustered using shared virtual disks
 
 ## Alerting and reporting
 
-The Zerto Self-Service Portal provides live real-time reporting of your solution and the current level of protection being provided to your VMs.
+The Zerto Self-Service Portal (ZSSP) provides live real-time reporting of your solution and the current level of protection being provided to your VMs.
```
Comment (gsmith101) [inline articles/draas/draas-sco.md:128]: ZSSP

**Review by gsmith101 — COMMENTED** [review]
The Zerto Self-Service Portal (ZSSP) should be defined at its first use and then only referred to using the acronym for the rest of the doc. :)

File: articles/draas/draas-sco.md (hunk 2 above)
Comment (gsmith101) [reply articles/draas/draas-sco.md:128]: Remove "The Zerto Self-Service Portal (ZSSP)".

The ZSSP provides live real-time reporting......

File: articles/draas/draas-sco.md (hunk 1 above)
Comment (suehighmoor) [reply articles/draas/draas-sco.md:116]: Fixed - Zerto Self-Service Portal is first mentioned at line 74 in the Frequency and Timing section. Added the acronym there and used the acronym throughout the rest of the article

File: articles/draas/draas-sco.md (hunk 2 above)
Comment (suehighmoor) [reply articles/draas/draas-sco.md:128]: Fixed - Zerto Self-Service Portal is first mentioned at line 74 in the Frequency and Timing section. Added the acronym there and used the acronym throughout the rest of the article

**Comment by suehighmoor** [comment]
@gsmith101 - I've addressed your comment - are you happy to approve now?

**Review by gsmith101 — APPROVED** [review]
Looks good to me.

**Pull request merged**